app/views/hub/clients/index.html.erb

```diff
@@ -4,8 +4,20 @@
 <% end %>
 <% content_for :card do %>
   <section class="slab clients-header">
-    <h1> <%= @page_title %> </h1>
     <div>
+      <h1> <%= @page_title %> </h1>
+      <div class="text--no-wrap quick-filters">
+        <label>Quick Filters:</label>
+        <% ClientSortable::QUICK_FILTERS.each do |(filter_value, filter_label)| %>
+          <% selected = filtering_only_by?(:last_contact, filter_value) %>
+          <%= link_to selected ? hub_clients_path(clear: true) : hub_clients_path(last_contact: filter_value), class: "button button--quick-filter  #{selected ? "selected" : ""}" do %>
+            <%= filter_label %>
+            <i class="clear-filter icon-close"></i>
```

Comment:
My component-oriented mind makes me think this would make sense as like a partial where we pass in the data it needs, or maybe even to style the icon as an ::after element instead of having the css for the icon right here. Something feels weird about building such a complex little UI element right on the index page because then it's not really reusable as a UI element